homeassistant/components/cambridge_audio/media_player.py

```diff
@@ -142,6 +142,12 @@ def media_title(self) -> str | None:
     @property
     def media_artist(self) -> str | None:
         """Artist of current playing media, music track only."""
+        if (
+            self.client.play_state.metadata.artist is None
```

Comment:
Small nitpick.

```suggestion
            not self.client.play_state.metadata.artist
```
This will make it also return true if the artist is an empty string.